src/components/Footer.jsx

```diff
@@ -30,7 +35,7 @@ const Footer = () => {
               >
                 Contact Us
               </a>
-              <a href="#">Privacy Policy</a>
+              <a href="#" onClick = {showModal}>Privacy Policy</a>
```

Comment:
TODO(@Jasonwei1106): instead change this to:

```javascript
<a href="/" onClick = {showModal}>Privacy Policy</a>
```
and inside `showModal` do evt.preventDefault i.e. stop default behaviour on click and return false;